tests/conftest.py

```diff
@@ -139,6 +140,30 @@ def executing_kernel() -> Generator[Kernel, None, None]:
     mocked.teardown()
 
 
+@pytest.fixture
+def temp_marimo_file() -> Generator[str, None, None]:
```

Comment:
PyTest has a built-in fixture that provides (and cleans up) a temporary directory: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
this one specifically write marimo content